lib/init.lua

```diff
@@ -1,9 +1,11 @@
 local Store = require(script.Store)
 local createReducer = require(script.createReducer)
 local combineReducers = require(script.combineReducers)
+local loggerMiddleware = require(script.loggerMiddleware)
 
 return {
 	Store = Store,
 	createReducer = createReducer,
 	combineReducers = combineReducers,
+	logger = loggerMiddleware,
```

Comment:
Should we name logger `loggerMiddleware` in the public API?

Reply:
I figured just calling it `logger` was okay, but I think `loggerMiddleware` is a better name in hindsight - it makes it clear that it's a middleware function, not something else.